rust/extractor/src/rust_analyzer.rs

```diff
@@ -51,7 +51,7 @@ impl<'a> RustAnalyzer<'a> {
         project: &ProjectManifest,
         config: &CargoConfig,
     ) -> Option<(RootDatabase, Vfs)> {
-        let progress = |t| (log::trace!("progress: {}", t));
+        let progress = |t| (tracing::trace!("progress: {}", t));
```

Comment:
very minor nit: again we can drop the `tracing` namespace here provided we also pull `trace` in the `use` above